metadata-ingestion/src/datahub/ingestion/source/powerbi/powerbi-lexical-grammar.rule

```diff
@@ -435,6 +435,8 @@ optional_parameter: "optional" WS_INLINE parameter
 each_expression:    "each" WS_INLINE each_expression_body
 
 each_expression_body:   function_body
+                |       invoke_expression NEWLINE? WS_INLINE? "+" NEWLINE? WS_INLINE? invoke_expression
```

Comment:
What are these changes for?

Reply:
`'let\n    Source = Databricks.Catalogs("adb-123.azuredatabricks.net", "/sql/1.0/endpoints/12345dc91aa25844", [Catalog=null, Database=null]),\n    hive_metastore_Database = Source{[Name="hive_metastore",Kind="Database"]}[Data],\n    sandbox_revenue_Schema = hive_metastore_Database{[Name="sandbox_revenue",Kind="Schema"]}[Data],\n    public_consumer_price_index_Table = sandbox_revenue_Schema{[Name="public_consumer_price_index",Kind="Table"]}[Data],\n    #"Renamed Columns" = Table.RenameColumns(public_consumer_price_index_Table,{{"Country", "country"}, {"Metric", "metric"}}),\n #"Inserted Year" = Table.AddColumn(#"Renamed Columns", "ID", each Date.Year([date_id]) + Date.Month([date_id]), Text.Type),\n #"Added Custom" = Table.AddColumn(#"Inserted Year", "Custom", each Text.Combine({Number.ToText(Date.Year([date_id])), Number.ToText(Date.Month([date_id])), [country]})),\n    #"Removed Columns" = Table.RemoveColumns(#"Added Custom",{"ID"}),\n    #"Renamed Columns1" = Table.RenameColumns(#"Removed Columns",{{"Custom", "ID"}}),\n #"Filtered Rows" = Table.SelectRows(#"Renamed Columns1", each ([metric] = "Consumer Price Index") and (not Number.IsNaN([value])))\nin\n    #"Filtered Rows"',`

The for each part in this query. U can check the slack convo with mohdsiddique  here. 
https://datahubspace.slack.com/archives/C027JMRUSQJ/p1678096601528009?thread_ts=1673841991.937759&cid=C027JMRUSQJ